apps/ui/src/composables/useSharing.ts

```diff
@@ -0,0 +1,82 @@
+import { FunctionalComponent } from 'vue';
+import { Proposal } from '@/types';
+import ICFarcaster from '~icons/c/farcaster';
+import ICLens from '~icons/c/lens';
+import ICX from '~icons/c/x';
+
+export type ShareableType = 'proposal' | 'user';
+type SocialNetwork = 'x' | 'lens' | 'farcaster';
+
+const HASH_TAG = 'Snapshot';
+
+const SOCIAL_NETWORKS: {
+  id: SocialNetwork;
+  name: string;
+  icon: FunctionalComponent;
+}[] = [
+  { id: 'x', name: 'X', icon: ICX },
+  { id: 'lens', name: 'Hey', icon: ICLens },
+  { id: 'farcaster', name: 'Warpcast', icon: ICFarcaster }
```

Comment:
Icon belong to farcaster and name is `Wrapcast` 

<img width="294" alt="Untitled 2" src="https://github.com/user-attachments/assets/6ac793cb-7e3f-407e-910e-144b38a24fe5">

maybe we should update logo?

Reply:
What do you think @bonustrack ? Should we share on farcaster (the protocol), or on warpcast (and change the icon) ?

Reply:
Yea actually we can change it and say Share on Lens, and Share on Farcaster